system-int-test/test/cmr/system_int_test/ingest/deleted_granules_index_test.clj

```diff
@@ -0,0 +1,63 @@
+(ns cmr.system-int-test.ingest.deleted-granules-index-test
+  "When a granule is deleted, a document is indexed in the deleted-granules index.
+   This namespace tests that functionality"
+  (:require
+   [cheshire.core :as json]
+   [clojure.edn :as edn]
+   [clojure.java.io :as io]
+   [clojure.string :as str]
+   [clojure.test :refer :all]
+   [cmr.common.mime-types :as mt]
+   [cmr.common.util :as u :refer [are3]]
+   [cmr.indexer.data.index-set :as index-set]
+   [cmr.indexer.system :as indexer-system]
+   [cmr.metadata-db.services.concept-service :as concept-service]
+   [cmr.metadata-db.int-test.utility :as mdb-test-util]
+   [cmr.system-int-test.data2.core :as d]
+   [cmr.system-int-test.data2.granule :as dg]
+   [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
+   [cmr.system-int-test.utils.dev-system-util :as dev-sys-util]
+   [cmr.system-int-test.utils.index-util :as index]
+   [cmr.system-int-test.utils.ingest-util :as ingest]
+   [cmr.system-int-test.utils.metadata-db-util :as mdb]
+   [cmr.system-int-test.utils.search-util :as search]
+   [cmr.umm.umm-granule :as umm-g]))
+
+(use-fixtures :each (ingest/reset-fixture {"provguid1" "PROV1"}))
+
+(defn check-index-for-deleted-granule
+  "Check elastic search deleted-granules index from related deleted granule entry,
+   Returns true if document exists, false if it does not."
+  [deleted-granule-id]
+  (index/doc-present? "deleted-granules" "deleted-granule" deleted-granule-id))
+
+(deftest deleted-granules-test
```

Comment:
So when you delete a collection, it should cascade to the granules, right? And we'd populate the deleted granule index? If so, can we add a test for that scenario?